main/autogen/autoloader.h

```diff
@@ -123,7 +127,7 @@ class DefTreeBuilder {
                              NamedDefinition);
 
     static DefTree merge(const core::GlobalState &gs, DefTree lhs, DefTree rhs);
-    static void markPackages(const core::GlobalState &gs, DefTree &root);
+    static void markPackages(const core::GlobalState &gs, DefTree &root, const AutoloaderConfig &);
```

Comment:
Can we add a variable name here for documentation?

```suggestion
    static void markPackages(const core::GlobalState &gs, DefTree &root, const AutoloaderConfig &autoloaderCfg);
```

Reply:
fixed